doc/active_docs/account_management_api.json

```diff
@@ -7836,10 +7836,6 @@
                     "type": "string",
                     "description": "System Name of the method. This is the name used to report API requests with the Service Management API. If blank, a system_name will be generated for you from the friendly_name parameter"
                   },
-                  "unit": {
```

Comment:
This needs to be removed from the "required" list also.

Reply:
@mayorova done